apps/datahub/src/app/search/search-filters/search-filters.component.html

```diff
@@ -22,6 +22,7 @@
   </div>
   <div
     class="flex flex-row items-start gap-4 -ml-[2px] mt-2 justify-betwwen w-56"
+    [ngClass]="{ 'w-60': displayCount === searchConfig.length }"
```

Comment:
Couldn't we just raise the `w-56` above to `w-60`? 

Btw, I'm also noticing that this area of the search filters is not responsive at all, but this seems to be out of scope for this PR I guess, as responsiveness has not yet been a real requirement.

Reply:
Yes, we totally can, I made the change!